tests/kernel/mem_slab/mslab_api/src/test_mslab_api.c

```diff
@@ -289,6 +289,14 @@ void test_mslab_used_get(void)
  */
 void test_mslab_pending(void)
 {
+	if (!IS_ENABLED(CONFIG_MULTITHREADING)) {
+		return;
```

Comment:
Would it be better if we put ztest_test_skip() here? It means test case is skipped, not pass.

Reply:
Hi, @enjiamai , put ztest_test_skip() here will trigger a build error which is the same as #40020, so it's better to be a return here.